gradle.properties

```diff
@@ -76,10 +76,10 @@ repeat = 100
 
 org.gradle.caching = true
 org.gradle.configureondemand = false
-org.gradle.daemon = true
+org.gradle.daemon = false
 org.gradle.jvmargs = -Xmx3g
 org.gradle.parallel = true
-#org.gradle.workers.max = 3
+org.gradle.workers.max = 3
 
```

Comment:
@udo I think these settings are intended for your machine, as opposed to defaults for everyone. Can you drop this from the PR, and consider moving your preferred values into `${HOME}/.gradle/gradle.properties`

Reply:
Yes, sorry..